playbooks/configure_ovs_dpdk.yml

```diff
@@ -5,6 +5,12 @@
   strategy: linear
   gather_facts: "{{ gather_facts | default(false) }}"
   tasks:
+    - name: Install openvswitch package
+      become: true
+      ansible.builtin.dnf:
+        name: openvswitch
+        state: present
+
```

Comment:
would you want to use edpm_ovs/tasks/install.yml here instead? The other task that would be done is to ensure that the openvswitch is started. Does dpdk require that?

Reply:
using  edpm_ovs/tasks/install.yml in configure ovs dpdk playbook, will create the enabled and running ovs part duplication also.
Configure ovs dpdk is require only openvswitch package installation and not to be enabled and running.
configure network is common for all feature and configure ovs dpdk is specific to only ovs dpdk.
better we can add only openvswitch installation here and we can avoid redundant ovs configuration and enabled part if we are not using ovs install.yml here.